app/Models/Foundation/Summit/Registration/SummitRegistrationInvitation.php

```diff
@@ -321,8 +322,15 @@ public function clearOrder()
 
     /**
      * @param SummitTicketType $ticketType
+     * @throws ValidationException
      */
     public function addTicketType(SummitTicketType $ticketType){
+        if ($ticketType->getAudience() != SummitTicketType::Audience_With_Invitation) {
+            throw new ValidationException
+            (
+                "Ticket type {$ticketType->getId()} must have audience attribute \"With Invitation\" to be added to this invitation {$this->getId()}"
```

Comment:
please add a final point (.) on the setence

Reply:
done